src/transformers/models/qwen2_vl/image_processing_qwen2_vl.py

```diff
@@ -192,13 +196,14 @@ def __init__(
         self.patch_size = patch_size
         self.temporal_patch_size = temporal_patch_size
         self.merge_size = merge_size
-        self.size = {"min_pixels": min_pixels, "max_pixels": max_pixels}
+        self.size = size if size is not None else {"min_pixels": min_pixels, "max_pixels": max_pixels}
```

Comment:
One last thing - if `size` can now be set in the input i.e. the values of min_pixels and max_pixels in the size dict are taken as precedence over the `min_pixels` and `max_pixels` input arguments, then we should pop `max_pixels` and `min_pixels` in the `to_dict` method so that only `size` is saved out into the config

Reply:
hmm, that's a good point. But I think the priority should be given to the `max_pixel` and `min_pixels` rather than size dict, which is the default setting for qwen2-vl

Reply:
OK - what I'd recommend is just not accepting the `size` argument and sticking with `min_pixels` and `max_pixels` then